components/ProjectModal/styles.ts

```diff
@@ -79,3 +83,55 @@ export const closeButtonStyles: CSSProperties = {
 export const additionalInfoStyles: CSSProperties = {
   width: '315px',
 };
+
+// STYLED COMPONENTS
+export const AllKDMS = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 300px;
+  height: 203px;
+  gap: 12px;
+`;
+
+export const KeyDevelopmentMilestone = styled.div<{ status: boolean }>`
+  padding-top: 8px;
+  padding-left: 15px;
+  width: 127px;
+  height: 30px;
+  flex-shrink: 0;
+  border-radius: 12px 4px 4px 4px;
```

Comment:
I can't see the actual output but flagging this asymmetry in case this was an accident.